L1Trigger/TrackFindingTracklet/src/PurgeDuplicate.cc

```diff
@@ -245,12 +245,7 @@ void PurgeDuplicate::execute(std::vector<Track>& outputtracks, unsigned int iSec
             continue;
           }
           // If the number of tracks able to be compared is more than the number of comparison modules, continue
-          if (CM > settings_.numTracksComparedPerBin()) {
-            continue;
-          }
           for (unsigned int jtrk = itrk + 1; jtrk < numStublists; jtrk++) {
-            if (itrk >= settings_.numTracksComparedPerBin())
-              continue;
```

Comment:
It was these lines 252-253 that was the cause of the duplicate fraction increase.